source/docs/articles/wordpress/hyperdb.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Scaling WordPress with MySQL Replication
+description: Learn how to distribute requests to replica MySQL databases on WordPress using HyperDB.
+---
+
+Replica MySQL database configuration via the [HyperDB](https://wordpress.org/support/plugin/hyperdb) plugin is available only for WordPress sites on the [Elite service level](/resources/elite-plan-overview). [Contact sales](/pantheon-elite-plans) to inquire about our Elite plans.
```

Comment:
This should be an alert box. It should read:

"Replica MySQL databases are available for sites on the [Elite service level](/resources/elite-plan-overview). [Contact sales](/pantheon-elite-plans) to inquire about our Elite plans."